mountpoint-s3/Cargo.toml

```diff
@@ -48,6 +48,8 @@ tracing = { version = "0.1.41", features = ["log"] }
 tracing-log = "0.2.0"
 tracing-subscriber = { version = "0.3.19", features = ["env-filter"] }
 
+statistical = { version = "1.0", optional = true }
```

Comment:
This crate is not maintained, so I'm not super fond of using it. Maybe we can use something that has seen more activity, or just calculate our own mean since that's all we're using.

If we eliminate the dependency, it also simplifies the optional dependency issue.

Reply:
I removed the dependency. 👍